### ISSUE TITLE
More flexibility on USB/Serial interface design could be nice

### ISSUE THREAD

**Issue by MrDham** [GaudiLabs/OpenThereminV4#1]
Hello Urs, 

I had a look at this new theremin schematic. It is a very nice idea to merge arduino and schield board in a single one. 

I hope that I don't bother you in a too much early or too much late stage of the development, I have just one comment: 

As CH340G can't be re-programmed into a MIDI USB port, in order to allow implementation of some additionnal material for a MIDI interface (MIDI DIN connector, ... ), an additionnal free solder pad on TXD could be very nice to have. 

Regards

**Comment by MrDham** [comment]
It seem that this issue is fixed, isn't it ?

In most recent design, I can see that a new pad is connected to RXD of CH340G wich is equivalent to TXD of ATMEGA 328P -> correct :+1:

**Comment by GaudiLabs** [comment]
Yes, we can now do MIDI over serial and real MIDI connector to TX pad.
